### PR TITLE
Refactoring Notifications::NewComment::Send

### PR DESCRIPTION
**Pull request by araslanov-e**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/services/notifications/new_comment/send.rb [hunk 1]
```diff
@@ -1,7 +1,12 @@
+# frozen_string_literal: true
+#
 # send notifications about the new comment
+
 module Notifications
   module NewComment
     class Send
+      attr_reader :comment
```
Comment (lightalloy) [inline app/services/notifications/new_comment/send.rb]: Do we really need a public `attr_reader`?

Comment (araslanov-e) [reply app/services/notifications/new_comment/send.rb]: Maybe not. I can add `private :comment`

Comment (lightalloy) [reply app/services/notifications/new_comment/send.rb]: This line was under `private` section, so I was wondering why was it moved.

Comment (araslanov-e) [reply app/services/notifications/new_comment/send.rb]: Yeah, missed it. Just more readable

Comment (rhymes) [reply app/services/notifications/new_comment/send.rb]: @araslanov-e let me know when you changed this, I'll then approve the PR

Comment (araslanov-e) [reply app/services/notifications/new_comment/send.rb]: @rhymes done

**Comment by araslanov-e** [comment]
@lightalloy done!

**Pull request merged**